src/models/plugins/IValidateProofResponse.ts

```diff
@@ -0,0 +1,6 @@
+export interface IValidateProofResponse {
```

Comment:
Consider `IValidatePoiResponse`.

Reply:
Named it IValidateProofResponse to match the inx-poi model but makes more sense to change ti to IValidatePoiResponse be consistent.